src/irr/video/COpenGLSpecializedShader.cpp

```diff
@@ -19,7 +19,6 @@ static void specialize(spirv_cross::CompilerGLSL& _comp, const asset::ISpecializ
     for (const auto& sc : sconsts)
     {
         spirv_cross::SPIRConstant& val = _comp.get_constant(sc.id);
-        val.specialization = false; // despecializing. If spec-data is not provided, the spec constant is left with its default value. Default value can be queried through introspection mechanism.
```

Comment:
what's this about?

Reply:
apparently it's not needed and spirv-cross didn work right with it. Thats why local_size_x/y/y_id didnt work actually